lib/getAccessorByteStride.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+var byteLengthForComponentType = require('./byteLengthForComponentType');
+var numberOfComponentsForType = require('./numberOfComponentsForType');
+
+module.exports = getAccessorByteStride;
+
+/**
+ * Returns the byte stride of the provided accessor.
+ * If the byteStride is 0, it is calculated based on type and componentType
+ *
+ * @param {Object} [accessor] The accessor.
+ * @returns {Number} The byte stride of the accessor.
+ */
+function getAccessorByteStride(accessor) {
+    if (accessor.byteStride > 0) {
+        return accessor.byteStride;
+    }
+    else {
```

Comment:
Drop the `else`.

See "Do not use an unnecessary else block at the end of a function" here: https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/CodingGuide#functions